apps/omg_api/test/integration/happy_path_test.exs

```diff
@@ -96,4 +96,14 @@ defmodule OMG.API.Integration.HappyPathTest do
 
     assert {:error, {_, "Internal error", "utxo_not_found"}} = Client.call(:submit, %{transaction: token_tx})
   end
+
+  defp wait_for_current_child_block(blknum, timeout \\ 10_000, contract \\ nil) do
```

Comment:
this looks quite similar to `wait_for_root_chain_block` from `DepositHelper`, shouldn't they be in the same place ? and